test/main/refresh-periodic-test

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+. libtest.sh
+. libgit.sh
+. "$source_dir/util.sh"
+
+export LINES=5
+
+steps '
+	:save-display main-with-unstaged.screen
+
+	:exec @git add a
+	:save-display main-after-add-a.screen
+
+	:exec @git commit -m "Commit changes"
+	:save-display main-after-commit.screen
+
+	:exec @git add b.c
+	:exec @git commit -m "Another change: hello"
+	:exec @sleep 2
```

Comment:
I need an advice about test/main/refresh-periodic-test: as it is, I am not sure that it correctly checks that the automatic refresh works. Sleeping for several seconds through exec, seems to block the whole program, which is not the same as tig being idle for this time. Is there a better way to ensure that tig will just simply sit idle for a while?